pkg/log/log.go

```diff
@@ -0,0 +1,91 @@
+package log
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Logger wraps of logrus logging into a file
+type Logger struct {
+	file  *os.File
+	Entry *logrus.Entry
+}
+
+// Close closes loggers and its log file
+func (logger *Logger) Close() error {
+	if err := logger.file.Sync(); err != nil {
+		logrus.Errorf("Fail to sync file %s, error: %s", logger.file.Name(), err)
+		return err
+	}
+
+	if err := logger.file.Close(); err != nil {
+		logrus.Errorf("Fail to close file %s, error: %s", logger.file.Name(), err)
+		return err
+	}
```

Comment:
nit: `Close` should cause a `Sync` internally, no need to call it here

Reply:
Thanks for the info